.github/workflows/openapi-generator.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will regenerate the client library API bindings based on the
+# public OpenAPI spec.
+
+name: OpenAPI Generator
+
+on:
+  workflow_dispatch:
+    inputs: null
+  schedule:
+    - cron: "0 * * * *"
```

Comment:
Does GitHub Action support webhooks, e.g. on file change?

Reply:
it gets really complicated trying to trigger actions across repos. it is possible but its more work than i want to put into it right now

Reply:
I think we can start with this. I don't think there is a need for real time PR updates to the api bindings. The level of fidelity we get by checking once per hour is more than adequate for now. Plus we can run them manually if we don't want to wait for the automation. 

If we wanted to trigger some action on file change it would involve

1. triggering the actions that generate the clients from the front repo
2. cloning the front repo into the actions that generate the clients

Both of these things are made more difficult by front being a private repo (we'd need to pass around API tokens everywhere).

I like this approach also because front doesn't have to know about any of the clients that use the spec file.